packages/wallets/src/features/cfd/flows/OtherCFDs/Dxtrade/AvailableDxtradeAccountsList/AvailableDxtradeAccountsList.tsx

```diff
@@ -14,7 +14,11 @@ const AvailableDxtradeAccountsList: React.FC = () => {
 
     return (
         <TradingAccountCard
-            leading={<div className='wallets-available-dxtrade__icon'>{PlatformDetails.dxtrade.icon}</div>}
+            leading={
+                <div className='wallets-available-dxtrade__icon' data-testid='dt_icon-dxtrade'>
```

Comment:
minor nitpick to follow the naming standard:
```suggestion
                <div className='wallets-available-dxtrade__icon' data-testid='dt_icon_dxtrade'>
```
Reference: https://deriv-group.slack.com/archives/C3AHMSM3P/p1702363978866289